src/test.java

```diff
@@ -1,28 +1,36 @@
 import java.util.ArrayList;
+import java.util.Scanner;
 
 public class test {
     public static void main(String[] args) {
 
         UsersBook usersBook = new UsersBook();
-        User user1 = new User("dsfsd","fdsfsf",25);
-        User user2 = new User("dsfsd","fdsfsf",4);
-
-        usersBook.addUser(user1);
-        usersBook.addUser(user2);
-
-        ArrayList<User> users = new ArrayList<>(usersBook.findAll());
-
-        for (User user:users) {
-            System.out.println(user);
+        Scanner scanner = new Scanner(System.in);
+        CommandInfoList commandInfoList = new CommandInfoList();
+
+        AddCommand addCommand = new AddCommand();
+        GetAllUsersCommand getAllUsersCommand = new GetAllUsersCommand();
+        GetUserForIdCommand getUserForIdCommand = new GetUserForIdCommand();
+
+        Context context = new Context();
+
+        while (true) {
+            System.out.println();
+            commandInfoList.printCommandInfoList(new ArrayList<>(commandInfoList.initialization()));
+            String inputCommand = scanner.nextLine();
+            if ("add".equals(inputCommand)) {
```

Comment:
But if you will add new command or remove existing, you will modify main method( add new if statement or remove)
Idea was to keep main method unmodifiable